include/clang/Serialization/ASTBitCodes.h

```diff
@@ -1322,7 +1322,6 @@ namespace clang {
       EXPR_OBJC_ARRAY_LITERAL,
       EXPR_OBJC_DICTIONARY_LITERAL,
 
-    
       /// \brief An ObjCEncodeExpr record.
```

Comment:
Thanks, I will revert the white space. And I suppose that there is more place like this. I will revert them too.!